lib/balrog/generators/view_generator.rb

```diff
@@ -0,0 +1,26 @@
+class Balrog::ViewGenerator < Rails::Generators::Base
+
+  desc "Copies the Balrog gate view and layout into your application, where you can edit and style them."
+
+  def copy_gate_view
+    gate_view = File.open(
+      File.join(__dir__, '../../../', 'app/views/balrog/gate.html.erb'),
```

Comment:
I don't think this style works on Windows, but I'm not sure. Did you adapt this from somewhere else?

Reply:
[Devise](https://github.com/plataformatec/devise/blob/14863ba4c92cd9781a961be0486f0ea7dfe84144/lib/generators/devise/views_generator.rb#L75) do a similar thing.

Reply:
Thx